### PR TITLE
Save host telemetries as batch

### PR DESCRIPTION
**Pull request by fabriziosestito**
This PR changes the host telemetry handler to support batch of host telemetries rather than one at once.

### CODE REVIEW

**Review by nelsonkopliku — APPROVED** [review]
:top:

File: server/server.go [hunk 1]
```diff
@@ -45,17 +45,20 @@ func hostTelemetryHandler(adapters ...StorageAdapter) func(w http.ResponseWriter
 			return
 		}
 
-		var hostTelemetry HostTelemetry
-		err := json.NewDecoder(r.Body).Decode(&hostTelemetry)
+		var hostTelemetries []*HostTelemetry
```
Comment (dottorblaster) [inline server/server.go]: Why don't we call this `hostTelemetryEntries`?

**Review by dottorblaster — CHANGES_REQUESTED** [review]
One nitpick, then we're ready to merge

File: server/influxdb.go [hunk 2]
```diff
@@ -21,21 +22,29 @@ func NewInfluxDB(url string, token string, org string, bucket string) *InfluxDB
...
-		AddField("total_memory_mb", h.TotalMemoryMB).
-		AddField("cloud_provider", h.CloudProvider).
-		SetTime(h.Time)
-
-	return writeAPI.WritePoint(context.Background(), p)
+	var err error
+	for _, t := range h {
+		p := influxdb2.NewPointWithMeasurement(hostTelemetryMeasurement).
+			AddTag("agent_id", t.AgentID).
+			AddTag("agent_id", t.InstallationID).
```
Comment (arbulu89) [inline server/influxdb.go]: This should be `installation_id`, right?

File: server/server_test.go [hunk 3]
```diff
@@ -43,14 +43,25 @@ func TestHostTelemetryHandler(t *testing.T) {
...
-		AgentID:       "agent_id",
-		SLESVersion:   "15.2",
-		CPUCount:      16,
-		SocketCount:   32,
-		TotalMemoryMB: 32000,
-		CloudProvider: "azure",
-		Time:          time.Now(),
+	body, _ := json.Marshal([]*HostTelemetry{
+		{
+			AgentID:       "agent_id",
```
Comment (arbulu89) [inline server/server_test.go]: `InstallationID` missing in the test

**Review by arbulu89 — CHANGES_REQUESTED** [review]
Hey @fabriziosestito,
A type and a minor improvement in a test commented

**Review by dottorblaster — APPROVED** [review]
LGTM :rocket:

**Review by arbulu89 — APPROVED** [review]

**Pull request merged**